docs/probe.md

```diff
@@ -0,0 +1,177 @@
+# Probes and Meters: Examples for Adding to Code
+For an introduction to `Probe`s and `Meter`s, please refer to [Measurement Overview](./metrics.md).
+
+## Context
+To monitor particular aspects of an `armory run` session, the user needs to know the following factors:
+- What am I measuring?
+- When should I measure it?
+- Where should my custom monitoring script go?
+
+We assume the user is modifying code for models or attacks and wishes to use `Probe`s and `Meter`s to monitor certain variables within the code.
+
+Recall the steps for a minimal working example (in [Measurement Overview](./metrics.md)):
+1. Create `Probe` via `get_probe(name)`
+2. Place `Probe` actions
+3. Create `Meter` with processing functions that take input from created `Probe`
+4. Connect `Meter` to `Hub` via `get_hub().connect_meter(meter)`
+
+The examples will show how each of these steps are accomplished.
+
```

Comment:
the two examples look good. Just to confirm, you did successfully run these correct?

Reply:
the examples run, but there was the relevant question asked before about what to expect for outputs, which will depend on what the `Probe` is monitoring...

Reply:
checkout f402f30d